### PR TITLE
docs: fix grammar and typos

### PR DESCRIPTION
**Pull request by anastasialanz**
This PR fixes typos, acronyms, and rewords some things in sections for all of the docs pages.

### CODE REVIEW

File: docs/pages/components/Stepper.mdx [hunk 1]
```diff
@@ -64,7 +64,7 @@ For more compact steppers, the additional descriptive information can be include
 
 <ComponentProps
   children={{
-    description: 'children to be rendered as the step\'s visible label (if not visible label is desired, use the "tooltip" prop instead)'
+    description: 'children to be rendered as the step\'s visible label (if a not visible label is desired, use the "tooltip" prop instead)'
```
Comment (scurker) [inline docs/pages/components/Stepper.mdx:67]:
```suggestion
    description: 'children to be rendered as the step\'s visible label (if no visible label is desired, use the "tooltip" prop instead)'
```

I think we probably meant to do this?

File: docs/pages/components/Scrim.mdx [hunk 2]
```diff
@@ -10,10 +10,10 @@ import { Scrim, Button } from '@deque/cauldron-react'
...
+A scrim is a component that is a semi-transparent overlay over the viewport to provide emphasis, focus, or draw attention to certain information. This is most commonly used as a backdrop for dialogs, modals, and other elements that need primary focus on the page.
 
 <Note>
-  It is recommended that a focus trap is used to contain keyboard focus within the element that has primary focus. [Alert](./Alert) and [Modal](./Modal) both use [react-focus-trap](https://www.npmjs.com/package/focus-trap-react) to manage focus. If `Scrim` is being used with other components, a focus trap may be needed to prevent certain keyboard accessibility issues.
+  It is recommended that a focus trap is used to contain keyboard focus within the element that has a primary focus. [Alert](./Alert) and [Modal](./Modal) both use [react-focus-trap](https://www.npmjs.com/package/focus-trap-react) to manage focus. If `Scrim` is being used with other components, a focus trap may be needed to prevent certain keyboard accessibility issues.
```
Comment (scurker) [inline docs/pages/components/Scrim.mdx:16]: This still reads weird to me, but I can't quite place my finger on how to word this better.

**Review by scurker — CHANGES_REQUESTED** [review]
Just 2 small things, thanks for putting this together!

File: docs/pages/components/Stepper.mdx (hunk 1 above)
Comment (anastasialanz) [reply docs/pages/components/Stepper.mdx:67]: I can switch it back if that's the case!

File: docs/pages/components/Scrim.mdx (hunk 2 above)
Comment (anastasialanz) [reply docs/pages/components/Scrim.mdx:16]: Grammarly was mad about this one. Writing about technical things is hard.

**Pull request merged**